README.md

```diff
@@ -98,6 +99,45 @@ Xcode is also able to display `TODO` and `FIXME` tags directly in the source nav
 
 Other conventional comment tags, such as `NOTE` are not recognized by Xcode.
 
+### Types ###
+
+Always use Swift native types when available. Swift offers a bridging to Objective-C so you can use them whenever you want.
+
+**Preferred:**
+
+```
+class myClass {
+...
+}
+```
+
+**Not preferred:**
+
+```
+@objc class myClass {
```

Comment:
To be more explicit, I'd put above this:

`Avoid subclassing NSObject unless it is required to implement an NSObjectProtocol type. Subclassing NSObject or using the @objc flag automatically creates an objective-c object that uses dynamic dispatch over the preferred static of Swift`